src/main/java/factionsystem/Subsystems/UtilitySubsystem.java

```diff
@@ -385,7 +385,7 @@ public void run() {
     				{
     					isOnline = player.isOnline();
     				}
-    				if (isOnline && main.getConfig().getBoolean("powerDecreases")
```

Comment:
This if statement checked to see if a player was online as one of the conditions for decreasing their power, meaning power decreases don't affect offline players like they should. I negated the isOnline condition.